osquery/core/init.cpp

```diff
@@ -116,6 +116,13 @@ CLI_FLAG(uint64,
          alarm_timeout,
          15,
          "Seconds to allow for shutdown. Minimum is 10");
+#ifdef OSQUERY_LINUX
```

Comment:
Does it make sense to put the flag globally?  Do we have other platform specific flags?

Reply:
There are definitely other cases (think of the platform specific events flags).